trytond/ir/cron.py

```diff
@@ -111,7 +117,7 @@ def view_attributes(cls):
                 }),
             ]
 
-    def compute_next_call(self, now):
+    def compute_next_call(self, now=datetime.datetime.now()):
```

Comment:
la valeur par défaut ne sera évaluée qu'une seul fois à l'import.

Reply:
@cedk oui effectivement.
donc  None par défaut et  `if not now: now = datetime.datetime.now()` ?

Reply:
Oui mais c'est mieux de tester explicitement `now is None` afin d'attraper les erreurs comme passer 0.